jsapi/src/main.js

```diff
@@ -778,39 +780,62 @@ lux.Map.prototype.addGPX = function(url) {
     return style[feature.getGeometry().getType()];
   };
 
-  this.addVector(url, new ol.format.GPX(), styleFunction);
+  this.addVector(url, new ol.format.GPX(), styleFunction, opt_options);
 };
 
 /**
  * Adds a KML file on the map
  * @param {string} url Url to the GPX file
+ * @param {luxx.VectorOptions=} opt_options Options.
  * @export
  */
-lux.Map.prototype.addKML = function(url) {
-  this.addVector(url, new ol.format.KML());
+lux.Map.prototype.addKML = function(url, opt_options) {
+  this.addVector(url, new ol.format.KML(), undefined, opt_options);
```

Comment:
It may have been easier to read with a single object for options, rather than 2 parameters…